src/components/refactor/dialogs/two-windings-transformer-creation/ratio-tap-changer-pane/ratio-tap-changer-pane-taps.js

```diff
@@ -0,0 +1,457 @@
+            },
+            {
+                label: intl
+                    .formatMessage({ id: 'DeltaResistance' })
+                    .toUpperCase(),
+                id: 'resistance',
+                dataKey: STEPS_RESISTANCE,
+                numeric: true,
+                editor: TableNumericalInput,
+            },
+            {
+                label: intl
+                    .formatMessage({ id: 'DeltaReactance' })
+                    .toUpperCase(),
+                id: 'reactance',
+                dataKey: STEPS_REACTANCE,
+                numeric: true,
+                editor: TableNumericalInput,
+            },
+            {
+                label: intl
+                    .formatMessage({ id: 'DeltaConductance' })
+                    .toUpperCase(),
+                id: 'conductance',
+                dataKey: STEPS_CONDUCTANCE,
+                numeric: true,
+                editor: TableNumericalInput,
+            },
+            {
+                label: intl
+                    .formatMessage({ id: 'DeltaSusceptance' })
+                    .toUpperCase(),
+                id: 'susceptance',
+                dataKey: STEPS_SUSCEPTANCE,
+                numeric: true,
+                editor: TableNumericalInput,
+            },
+            {
+                label: intl.formatMessage({ id: 'Ratio' }).toUpperCase(),
+                id: 'ratio',
+                dataKey: STEPS_RATIO,
+                numeric: true,
+                fractionDigits: 5,
+                editor: TableNumericalInput,
+            },
+        ];
+    }, [intl]);
+
+    const defaultCellRender = useCallback(
+        (rowData) => {
+            return (
+                <>
+                    <div
+                        component="span"
+                        variant="body"
+                        className={classes.tableCell}
+                        align={'right'}
+                    >
+                        {rowData.cellData}
+                    </div>
+                </>
+            );
+        },
+        [classes.tableCell]
+    );
+
+    const editableCellRender = useCallback(
+        (row) => {
+            const index = row.columnIndex;
+            const Editor = COLUMNS_DEFINITIONS[index].editor;
+            if (Editor) {
+                return (
+                    <div className={classes.tableCell}>
+                        <Editor
+                            disabled={disabled}
+                            key={row.dataKey + row.rowData.id}
+                            name={`${RATIO_TAP_CHANGER}.${STEPS}[${row.rowIndex}].${row.dataKey}`}
+                            columnDefinition={COLUMNS_DEFINITIONS[index]}
+                        />
+                    </div>
+                );
+            }
+            return defaultCellRender(row);
+        },
+        [COLUMNS_DEFINITIONS, classes.tableCell, disabled, defaultCellRender]
+    );
+
+    const generateTableColumns = () => {
```

Comment:
What's the point of this method, why not directly add cellRenderer in COLUMNS_DEFINITIONS ?